crates/flipperzero/src/subghz/device.rs

```diff
@@ -0,0 +1,193 @@
+//! Safe wrapper utilizing the API provided in `lib/subghz/devices/devices.h`
+
+use super::{error::SubGhzError, preset::SubGhzPreset};
+use core::{ffi::CStr, ptr::null_mut};
+use flipperzero_sys::{
+    subghz_devices_begin, subghz_devices_deinit, subghz_devices_end, subghz_devices_flush_rx,
+    subghz_devices_flush_tx, subghz_devices_get_by_name, subghz_devices_idle, subghz_devices_init,
+    subghz_devices_is_connect, subghz_devices_is_frequency_valid,
+    subghz_devices_is_rx_data_crc_valid, subghz_devices_load_preset, subghz_devices_read_packet,
+    subghz_devices_reset, subghz_devices_rx_pipe_not_empty, subghz_devices_set_frequency,
+    subghz_devices_set_rx, subghz_devices_set_tx, subghz_devices_sleep,
+    subghz_devices_write_packet, SubGhzDevice,
+};
+use uom::si::{frequency::hertz, u32::Frequency};
+
+pub struct SubGhz {
+    device: &'static SubGhzDevice,
+}
+
+impl SubGhz {
+    /// For the internal device use "cc1101_int" (From `lib/subghz/devices/cc1101_int/cc1101_int_interconnect.c`)
```

Comment:
A link would be useful, I think

Note that there is cool Markdown syntax for link in footers:

```suggestion
    /// For the internal device use [`cc1101_int`]
    ///
    /// [`cc1101_int`]: https://github.com/flipperdevices/flipperzero-firmware/blob/dev/lib/subghz/devices/cc1101_int/cc1101_int_interconnect.c
```